terraform/mysql/mysql-datadog.tf

```diff
@@ -0,0 +1,28 @@
+resource "aiven_mysql" "mysql" {
+  project      = var.project_name
+  cloud_name   = var.cloud_name
+  plan         = var.mysql_plan
+  service_name = "mysql-datadog"
+}
+
+resource "aiven_service_integration_endpoint" "datadog" {
+  project       = var.project_name
+  endpoint_name = "my-Datadog"
+  endpoint_type = "datadog"
+
+  datadog_user_config {
+    datadog_api_key = var.datadog_api_key
+    datadog_tags {
+      tag = "environment:dev"
```

Comment:
imo this should be a variable too, rather than hardcoded

Reply:
Made the required changes